public/views/index-template.html

```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    [[ if and .CSPEnabled .TrustedTypesEnabled .TestModeEnabled]]
+    <!-- Cypress overwrite CSP headers in HTTP requests, so this is required for e2e tests-->
+    <meta  http-equiv="Content-Security-Policy" content="require-trusted-types-for 'script';"/>
```

Comment:
I would prefer to have this content come from `content_security_policy_template`, but I need to figure out how to replace `$NONCE` and `$ROOT_URL`

Reply:
Agree, that would be preferable. As a first iteration, only checking the behavior for Trusted Types could be sufficient.